setup.py

```diff
@@ -31,5 +42,9 @@
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
     ],
-    python_requires='>=3.5'
+    python_requires='>=3.5',
+    project_urls={
```

Comment:
Add Fetch.AI website? Community site? May be worth checking with Maria or Chris.